packages/chargebee/package.json

```diff
@@ -11,7 +11,7 @@
   "bugs": {
     "url": "https://github.com/goodhood-eu/goodhood/issues"
   },
-  "version": "1.4.0",
+  "version": "1.4.1",
```

Comment:
Imo it should be v1.5.0 since tracking is a new functionality, not a bugfix!

Reply:
+1